python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -6694,31 +6693,12 @@ def test_dataframe_info_null_counts():
             -6.88918237,
             -7.00001,
         ],
-        [
-            1.987654321,
-            2.987654321,
-            3.987654321,
-            0.1221,
-            2.1221,
-            0.112121,
-            -21.1212,
-        ],
-        [
-            -1.987654321,
-            -2.987654321,
-            -3.987654321,
-            -0.1221,
-            -2.1221,
-            -0.112121,
-            21.1212,
-        ],
```

Comment:
Here I am just deleting tests that might be redundant. I am basing this off the assumption that all these different sets of numbers are not really increasing test coverage.

Reply:
For some values of rtol and atol, the deleted values seem to be testing the behavior of `isclose` at a granularity finer than float32 (requiring float64s for accuracy). That seems potentially important, and would deserve a code comment explaining the coverage that each parameter set adds. Instead of deleting these, we might reframe these parameters so that we don't test a whole matrix of all input values and all atol/rtol values (6x6x6x6 = 1296 tests), and only test certain pieces of the matrix that actually cover the important pieces of the behavior (comparing ints and floats, testing `isclose` at float64 precision, etc.).

Reply:
AFAICT the these tests were only covering `float64` both before and after the change. The dataframe and array constructors implicitly upcast the incoming data, so it's always getting compared as `float64`. Is your concern that we're not covering `float32` at all here?

Reply:
Sorry, I wasn't very clear. The issue isn't really about data types. It's about the rtol / atol precision requiring `float64` precision, which these changes no longer test adequately. The real-world cases of `isclose` I have seen use very tight tolerances (sometimes tighter than float32 precision, like `1e-08`). Currently, this PR removes the input data that is designed to test those cases of tight tolerances.

If you look at the `data1`/`data2` values like `1.987654321` and `1.9876543`, those pairs of input data are meant to be compared with the rtol/atol values of `1e-08` in the other set of parameters. If we remove the more-precise values here, we aren't getting good coverage of the tighter tolerance `1e-08`, which requires `float64` precision to get the correct results. By removing these pairings of parametrized values, this test would no longer fail if the `isclose` function were to erroneously cast its inputs to `float32`.

I agree that this test is grossly over-parametrized, but the deletions here are removing an important case to check.

Reply:
Let's just undo these changes and add a `TODO` that we can be more clever about parametrizing this particular test. The other changes in this PR give a more-than-meaningful improvement in test time and I don't think it's worth investing much more time over this single test at the moment.

Reply:
I agree that would be good. I can file an issue describing what I've discussed with @brandon-b-miller via Slack.

Reply:
Sounds good to me. Thank you, Bradley!

Reply:
Issue #10284 filed. I can put this in my backlog of things to do, or I can help someone else construct the specific cases I have in mind for test coverage.

Reply:
This change has been reverted